lib/hz2600/Kazoo/Api/Entity/AbstractEntity.php

```diff
@@ -177,19 +177,27 @@ public function fetch($append_uri = null) {
      * id then it will be created.
      *
      */
-    public function save($append_uri = null) {
+    public function save($append_uri = null, $method = "post") {
         if ($this->read_only) {
             throw new ReadOnly("The entity is read-only");
         }
 
         $id = $this->getId();
-        $payload = $this->getPayload();
+
```

Comment:
Sorry, but I am not fully understand your comment. If smb wants to use method patch, smb must explicit set $method = "patch", otherwise $method will be "post". Why should we work hard? Did you mean add function "save" with two parameters?